shop/models/notification.py

```diff
@@ -129,7 +129,7 @@ def order_event_notification(sender, instance=None, target=None, **kwargs):
     from shop.models.order import OrderModel
     from shop.serializers.order import OrderDetailSerializer
 
-    if not isinstance(instance, OrderModel):
+    if not isinstance(instance, OrderModel._materialized_model):
```

Comment:
in my opinion it would be cleaner if we would test against
```
from shop.models.order import BaseOrder

if not isinstance(instance, BaseOrder):
    ...
```

what do you think?

Presumably the cleanest solution would be to implement ``__isinstancecheck__`` on ``OrderModel``, so that ``isinstance(instance, OrderModel)`` can be used on a lazy object. Unfortunately my last attempt to do so was without any luck.